app/controllers/evaluation_forms_controller.rb

```diff
@@ -119,4 +121,14 @@ def handle_upate_permitted_params(permitted)
       permitted
     end
   end
+
+  def custom_success_flashes(evaluation_form)
+    flash[:custom_success_heading] = I18n.t("evaluation_form.success.heading")
+    flash.now[:custom_success_description] = "
```

Comment:
looks like `flash.now` is suppressing the description from the flash message after I save an update successfully, which redirects
![Screenshot 2025-03-04 at 3 08 06 PM](https://github.com/user-attachments/assets/d1611b0b-d02b-467d-9319-baf21a5b3223)